lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -73,7 +74,10 @@ def setup_course_and_user(
 
         if enroll_user:
             enrollment_mode = enrollment_mode or CourseMode.DEFAULT_MODE_SLUG
```

Comment:
Nit: move enrollment_mode variable into the else clause since that is the only place it is used.